src/containers/shared/components/Notification/index.tsx

```diff
@@ -1,27 +1,35 @@
 import { useEffect, useState } from 'react'
-import * as PropTypes from 'prop-types'
 import './styles.scss'
 
-const PRIMARY = 'primary'
-const VALID_USAGES = [
-  'default',
-  'success',
-  'warning',
-  'danger',
-  'dark',
-  'light',
-  'dark50',
-]
+type NotificationLevel = 'primary'
```

Comment:
If this is just a single string, why is this not a constant? (Or just included in the component as a hard-coded value?)

Reply:
That is a good question.  The previous prop type was `level: PropTypes.oneOf([PRIMARY])` so this is equivalent.  Currently it is the only valid option but could be expanded later via like `'primary' | 'secondary'`.